src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="/" method="get">
+      <div class="input-wrapper">
+        <input
+          class="input"
+          name="input"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="big"
+        >
+        <img
+          class="icon"
+          src="./images/Search.svg"
+          alt="search-icon"
+        >
+      </div>
+    </form>
+
+    <form action="/" method="get">
+      <div class="input-wrapper">
+        <input
+          class="input input--small"
+          name="input-small"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="small"
+        >
+        <img
+          class="icon icon-small"
```

Comment:
Follow the style code, if you have already started defining modifiers through a double "-" then continue to do it same way everywhere